### PR TITLE
[FIX] migration from 11.0

### PR DESCRIPTION
**Pull request by sergiocorato**
solve #492

### CODE REVIEW

File: product_contract/migrations/12.0.3.0.0/pre-migration.py [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
-            UPDATE product_template
-            SET recurring_invoicing_type = 'post-paid'
-            WHERE recurring_rule_type = 'monthlylastday'
-        """
-    )
+from openupgradelib import openupgrade
+from odoo.tools import parse_version
+
+
+@openupgrade.migrate()
```
Comment (sbejaoui) [inline product_contract/migrations/12.0.3.0.0/pre-migration.py]: you don't need `openupgrade` here

```suggestion
```

File: product_contract/migrations/12.0.3.0.0/pre-migration.py [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+def migrate(env, version):
```
Comment (sbejaoui) [inline product_contract/migrations/12.0.3.0.0/pre-migration.py]:
```suggestion
def migrate(cr, version):
```

**Review by sbejaoui — CHANGES_REQUESTED** [review]

**Review by sbejaoui — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Just tested and works :+1:

**Pull request merged**